pennylane/__init__.py

```diff
@@ -227,13 +230,21 @@ def circuit():
     raise DeviceError("Device does not exist. Make sure the required plugin is installed.")
 
 
-def grad(func, argnum=None):
+make_vjp = unary_to_nary(_make_vjp)
```

Comment:
What is `unary_to_nary`? :laughing:

Reply:
I don't know 😆 For some reason, you have to do this before you can use this function.